src/components/EditRelays.tsx

```diff
@@ -97,7 +95,6 @@ function RelayFormGroup({
   onSave,
 }: RelayFormGroupProps): JSX.Element {
   const [isEdit, setIsEdit] = useState<boolean>(false);
-  const relayRef = useRef<HTMLInputElement | null>(null);
```

Comment:
why don't you need this anymore?

Reply:
it's just unused code, it has nothing to do with the change. but if i had kept it, i would have needed to use `React.forwardRef` for the Wrapper like in `ReactQuillWrapper`